ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/ActionManager.java

```diff
@@ -39,9 +39,31 @@ public void remove(Uid id)
 		_allActions.remove(id);
 	}
 
-    public int getNumberOfInflightTransactions()
+	/**
+	 * <p>
+	 *     This method works out the number of in-flight transactions currently in
+	 *     the system.
+	 * </p>
+	 * <p>
+	 *     Note: The definition of in-flight (a.k.a. active) transactions can be
+	 *     found in the OMG OTS specification: "[Active is defined as] the state
+	 *     of a transaction when processing is in progress and completion of the
+	 *     transaction has not yet commenced." In addition to the OMG OTS
+	 *     definition, transactions that are marked as abort-only are not
+	 *     considered active, as they have not started the 2PC protocol.
+	 *     In Arjuna, these considerations boil down to the following: in-flight
```

Comment:
I wonder if phrasal verbs are generally understandable to non native speakers?

Reply:
Fair, I will change "boil down" :-)

Reply:
Thanks, in other less formal contexts, like blogs, emails, articles etc, such verbs can be nice, but I felt not so much in formal documentation.

Reply:
No problem. I used "are distilled into" instead. Thanks Michael